bucket/file-optimizer.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "14.10.2534",
+    "description": "File optimizer featuring lossless file size reduction",
+    "homepage": "https://nikkhokkho.sourceforge.io/static.php?page=FileOptimizer",
+    "license": "AGPL-3.0-or-later",
+    "url": "https://downloads.sourceforge.net/project/nikkhokkho/FileOptimizer/14.10.2534/FileOptimizerFull.7z.exe#/dl.7z",
+    "hash": "sha1:a6ba351d9f5cfbd647c64b37f2d4b26c3c2bfd4d",
+    "architecture": {
+        "32bit": {
+            "bin": [
+                [
+                    "FileOptimizer32.exe",
+                    "FileOptimizer"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "FileOptimizer32.exe",
+                    "File Optimizer"
+                ]
+            ]
+        },
+        "64bit": {
+            "bin": [
+                [
+                    "FileOptimizer64.exe",
+                    "FileOptimizer"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "FileOptimizer64.exe",
+                    "File Optimizer"
+                ]
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\FileOptimizer32.ini\")) { New-Item \"$dir\\FileOptimizer32.ini\" -Force | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\FileOptimizer64.ini\")) { New-Item \"$dir\\FileOptimizer64.ini\" -Force | Out-Null }"
+    ],
```

Comment:
- ...
- Why force?
     - You need force only if the file already exists, and if it already exists you do not need to do the persist check ...

```
'32', '64' | % { if Test-Path $pers\FileOptimizer$_.ini { New-Item | Out-Null  } }
```

Reply:
> * ...
> * Why force?
>   * You need force only if the file already exists, and if it already exists you do not need to do the persist check ...
It's my fault. `Force` is only required in copy-over method. I didn't edit it out.